README.md

```diff
@@ -369,6 +369,21 @@ will give a more in-depth description of the command and available options.
  - `autograder.cli.courses.assignments.submissions.fetch.course.attempts` -- Get all the most recent submissions (code and grading output) for an assignment.
  - `autograder.cli.courses.users.list` -- List all the users in a course.
 
+#### Proxy Submissions
+
+Instructors and TAs may need to accomodate a student's unique situations.
+The autograder server includes proxy submissions so course staff may assist students if necessary.
+Proxy submissions allow course staff to target a `proxy-user` and set a `proxy-time`.
```

Comment:
This is a bit not direct enough for my liking.
Something more direct may be better, like:
"The autograder allowed course staff to submit on behalf of students using "proxy" submissions."